Sources/swift-parser-cli/TerminalUtils.swift

```diff
@@ -0,0 +1,51 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Glibc)
+import Glibc
+#elseif os(Windows)
+import CRT
+#else
+import Darwin.C
+#endif
+
+#if os(Android)
+typealias FILEPointer = OpaquePointer
+#else
+typealias FILEPointer = UnsafeMutablePointer<FILE>
```

Comment:
Should this be named `FilePointer` to look a little more swift? It’s bad enough to have `FILE` scream at you in all caps.